addon/components/au-data-table/content.hbs

```diff
@@ -1,14 +1,14 @@
 {{! template-lint-disable table-groups }}
 <div class="au-c-data-table__wrapper">
-  <table class="au-c-data-table__table {{tableClass}}" ...attributes>
-    {{#if hasBlock}}
+  <table class="au-c-data-table__table {{this.tableClass}}" ...attributes>
+    {{#if (has-block)}}
       {{yield (hash
-          header=(component "au-data-table-content-header" enableSelection=enableSelection enableLineNumbers=enableLineNumbers data-table=data-table)
-          body=(component "au-data-table-content-body" content=content enableSelection=enableSelection enableLineNumbers=enableLineNumbers noDataMessage=noDataMessage onClickRow=(optional onClickRow) data-table=data-table)
+          header=(component "au-data-table-content-header" content=this.content enableSelection=this.enableSelection enableLineNumbers=this.enableLineNumbers data-table=this.data-table)
```

Comment:
Why does the `content` value need to be passed in? It seems it wasn't needed previously?

Reply:
Luckily no harm, but total miss of mine.